src/main/perf/LineFileDocs.java

```diff
@@ -471,6 +486,29 @@ static Document cloneDoc(Document doc1) {
     return doc2;
   }
 
+  private String getRandomWord(String body) {
+    Random random = ThreadLocalRandom.current();
```

Comment:
Is it possible to seed this `Random` consistently?  Perhaps not, since threads are involved.

Reply:
Since I am now getting random labels in a separate Python script, I guess this comment doesn't really apply anymore.